docs/csharp/whats-new/csharp-7-2.md

```diff
@@ -60,9 +62,19 @@ int binaryValue = 0b_0101_0101;
 
 ## _private protected_ access modifier
 
-Finally, a new compound access modifier: `private protected` indicates that a member may be
+A new compound access modifier: `private protected` indicates that a member may be
 accessed by containing class or derived classes that are declared in the same assembly. While `protected internal`
 allows access by derived classes or classes that are in the same assembly, `private protected`
 limits access to derived types declared in the same assembly.
 
 For more information see [access modifiers](../language-reference/keywords/access-modifiers.md) in the language reference.
+
+## Conditional `ref` expressions
+
+Finally, the conditional expression may produce a ref result instead of a value result. For example, you would write the following to retrieve a reference to one of two arrays:
```

Comment:
"reference to one of two arrays" -> "reference to the first element of one of two arrays"